api/HTMLMediaElement.json

```diff
@@ -2296,10 +2296,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "3.5"
+              "version_added": "9"
```

Comment:
Can you update `error_event` as well? This doesn't seem right, but maybe the event itself was fired from 3.5 but the event handler attribute was missing. http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8721 is a test for it, it should be easy to turn into a custom test if we have async tests.